### PR TITLE
[REFACTORING] User show un SRJ

### PR DESCRIPTION
**Pull request by tochman**
This is a refactoring of the way we show a user profile. Instead of rendering an entire page, we are showing a popup with the user info. It was created as part of writing up a course material guide about unobtrusive JS.

TODO: 
* Add cancel button or X in the corner

There's no PT story for this. 

ca_user_show_and_edit

### CODE REVIEW

File: app/javascript/css/tailwind.scss [hunk 1]
```diff
@@ -153,6 +153,24 @@
...
+    cursor: pointer;
+    /* display: inline-block; */
+    font-family: arial;
+    /* font-weight: bold; */
+    position: absolute;
+    top: 3px;
+    right: 3px;
+    font-size: 18px;
+    line-height: 30px;
+    width: 30px;
```
Comment (oliverochman) [inline app/javascript/css/tailwind.scss:169]: why did you not use the tailwind classes?

Comment (tochman) [reply app/javascript/css/tailwind.scss:169]: Because I had this one ready from before.

Comment (oliverochman) [reply app/javascript/css/tailwind.scss:169]: Okay

**Review by oliverochman — APPROVED** [review]
LGTM!

**Pull request merged**